_posts/1000-09-03-config-mgmt.md

```diff
@@ -0,0 +1,20 @@
+---
+date: 2018-04-30
+title: Ansible and Config Management
+categories:
+   - configuration managements
+   - ansible
```

Comment:
You should pick one category, or this will add all of these categories on the KB Site